### ISSUE TITLE
Should we show the number of download left instead of the initial limit value?

### ISSUE THREAD

**Issue by skjnldsv** [nextcloud/files_downloadlimit#77]
Currently we show a tooltip to show the number of downloads.
I think we should directly show the number of downloads left?

![image](https://user-images.githubusercontent.com/14975046/162450554-e4446f3b-d677-405c-ba62-7e3464031e0b.png)

cc @PVince81 @jancborchardt

**Comment by skjnldsv** [comment]
It makes sense because the remaining downloads is the current download limit. To show the initial download limit one week later to the customer does not deliver any new information to the user because he may remember the initial download limit or can calculate it with the download counter. It is more helpful to tell the user the current download limit to enable him to decide if he wants to increase it or not.

_Originally posted by @TSI-yogeshshejwadkar in https://github.com/nextcloud/files_downloadlimit/pull/69#discussion_r835071896_

**Comment by szaimen** [comment]
yes, makes sense to me 👍

**Comment by jancborchardt** [comment]
Yep, since it’s about a limit here and not analysis, the "how many left" is more important than how many downloads happened.
